Digipost.Api.Client.DataTypes/Appointment/AppointmentAddress.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Digipost.Api.Client.DataTypes
+{
+    public class AppointmentAddress
+    {
+        /// <inheritdoc />

```

Comment:
Waaat? Er dette en greie?  Megafett!!! Men vi har ikke noen dok?

Reply:
Hvordan fungerer denne med forskjellig antall parametere?

Reply:
Aner ikke, det var du som la til dette :D

Reply:
Ser imidlertid ikke ut til å ha noen effekt. Antar det egentlig er meninga å bruke på metoder som `override`r andre? Inherit lissom. Arv. You know.

Reply:
Så. Jeg fjerner. Men erre meninga at jeg skal skrive noe dok der?